apertium-uzb.uzb.lexc

```diff
@@ -90,17 +126,18 @@ Multichar_Symbols
 %<title%>                         !
 
 !! Tense / finite forms
-%<pres%>      ! for "жатыр", "тұр", "отыр" and "жүр"
+%<pres%>      ! for "yotir", "dir",..
```

Comment:
I don't think we need this in Uzbek.

Reply:
If that is what I understood it is, don't we still need this for old Uzbek when analyzing old texts, like from poetry?. 
Since you said so, I must be wrong, so I  commented it out.

Reply:
hi @elmurod1202 , 
Could you give us some old Uzbek text which contain "жатыр", "тұр", "отыр" and "жүр"?

Reply:
@elmurod1202, some examples of what you're talking about would be really helpful.